src/themes/dark-theme-colors.js

```diff
@@ -152,6 +152,17 @@ export default {
     menuFontDisabled: primitives.mono400,
     menuFontHighlighted: primitives.mono200,
     menuFontSelected: primitives.mono200,
+
+    // Pagination
+    paginationTriangleDown: primitives.mono100,
+
+    // Header navigation
+    headerNavigationFill: primitives.mono700,
+
+    // Tab
+    tabBarFill: primitives.mono1000,
+    tabColor: primitives.mono300,
+    tabColorActive: primitives.mono100,
```

Comment:
you can use `colorPrimary` instead of `tabColorActive`

Reply:
@chasestarr should be ready for another pass